docs/_docs/04_reference/cli-options.md

```diff
@@ -85,6 +85,8 @@ deployed them, then `plan-all` will fail as it will not be possible to resolve t
 `terraform_remote_state` data sources! Please [see here for more
 information](https://github.com/gruntwork-io/terragrunt/issues/720#issuecomment-497888756).
 
+**[NOTE]** Using `run-all` with `apply` or `destroy` silently adds the `-auto-approve` flag to the command line arguments passed to Terraform due to issues with shared `stdin` making individual approvals impossible. Please [see here for more information](https://github.com/gruntwork-io/terragrunt/issues/386#issuecomment-358306268)
```

Comment:
I think there might be a line-length issue. We default to trying to keep lines <=120 characters. Could you split these lines for these 3 changes?